examples/platform/silabs/efr32/rs911x/hal/efx_spi.c

```diff
@@ -184,6 +188,10 @@ static bool rx_dma_complete(unsigned int channel, unsigned int sequenceNo, void
     xSemaphoreGiveFromISR(spi_sem, &xHigherPriorityTaskWoken);
     portYIELD_FROM_ISR(xHigherPriorityTaskWoken);
 
+#if defined(SL_CATALOG_POWER_MANAGER_PRESENT)
+    sl_power_manager_remove_em_requirement(SL_POWER_MANAGER_EM1);
+#endif
```

Comment:
You have a add_requirement for receive and transmit dma but only 1 remove for rx dma complete. Is this normal?

Reply:
both of them have the callback of the rx_dma_complete. So adding that only one place is enough

Reply:
If it is the same function for both, it shouldn't have `rx` only in the name. Can you please change the function signature and theses details in the function comment?

Reply:
Added comments on the rx api
It is similar to the spidrv present in the gsdk

Reply:
Comment was not addressed. Function signature should take into account the fact that it is used for RX **AND** TX

Reply:
modified the function name to tx_rx_dma_complete